JankStatsSample/app/src/main/java/com/example/jankstats/tools/simulateJank.kt

```diff
@@ -39,7 +40,10 @@ fun simulateJank(
         }
 
         try {
+            // Make jank easier to spot in the profiler through tracing.
+            Trace.beginSection("Jank Simluation")
             Thread.sleep(delay)
+            Trace.endSection()
```

Comment:
nit, more kotlin friendly (if you accept it, it needs import)
```suggestion
            trace("Jank Simluation") {
                Thread.sleep(delay)
            }
```